services/telegram/service.go

```diff
@@ -0,0 +1,81 @@
+package telegram
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Service struct {
+	chat_id          string
+	url              string
+	global           bool
+	stateChangesOnly bool
+	logger           *log.Logger
+}
+
+func NewService(c Config, l *log.Logger) *Service {
+	return &Service{
+		chat_id:          c.ChatId,
+		url:              c.URL + c.Token + "/sendMessage",
+		global:           c.Global,
+		stateChangesOnly: c.StateChangesOnly,
+		logger:           l,
+	}
+}
+
+func (s *Service) Open() error {
+	return nil
+}
+
+func (s *Service) Close() error {
+	return nil
+}
+
+func (s *Service) Global() bool {
+	return s.global
+}
+func (s *Service) StateChangesOnly() bool {
+	return s.stateChangesOnly
+}
+
+func (s *Service) Alert(chatId, message string) error {
+	if chatId == "" {
+		chatId = s.chat_id
+	}
+	postData := make(map[string]interface{})
+	postData["chat_id"] = chatId
+	postData["text"] = message
+
+	var post bytes.Buffer
+	enc := json.NewEncoder(&post)
+	err := enc.Encode(postData)
+	if err != nil {
+		return err
+	}
+
+	resp, err := http.Post(s.url, "application/json", &post)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		type response struct {
+			Error string `json:"error"`
```

Comment:
As per the docs here is looks like errors are of the form 

```json
{
  "ok" : false,
  "description" : "error message here",
  "error_code": 576
}
```

Can you update the response struct accordingly?